docs: add website announcement bar feature highlight (#1104)
quarto-dev/quarto-web: docs/websites/website-tools.qmd

Note that links to figures should start with a `/` to work on each level of the website.

## Announcement Bar

![Captured view of the Quarto website, showcasing the dynamic announcement bar feature.](images/website-announcement.png){.border fig-alt="Screenshot showing an announcement bar on Quarto website with an 'info-circle' icon, a dismiss button, and a primary colour type. The content reads 'Alert - this is some information that you should pay attention to'."}

> **Website announcement bar** feature is a versatile tool designed to enhance user engagement and communication on your website.
> By adding a simple configuration to your `_quarto.yml` file, you can display a prominent, customizable announcement bar that grabs visitors' attention.
> This feature allows you to set an icon, make it dismissable, and even include formatted content like bold text.
> It's perfect for highlighting important information, such as alerts, promotions, or updates, directly to your audience.
> The announcement bar can be positioned to fit seamlessly within your site's layout (*e.g.*, `below-navbar` or `above-navbar`), ensuring the message is both impactful and integrated.

Reviewer: Some wording/ordering edits to make this a bit more concise:
```suggestion
Add an announcement to display a prominent, customizable bar at the top of your website that grabs visitors' attention.
It's perfect for highlighting important information, such as alerts, promotions, or updates.
You can set an icon, make it dismissable, and even include formatted content like bold text.
The announcement bar can be positioned to fit seamlessly within your site's layout (*e.g.*, `below-navbar` or `above-navbar`), ensuring the message is both impactful and integrated.
```
